### DISCUSSION TITLE
Would anyone like to add C3 to GitHub Linguist?

### DISCUSSION THREAD

**Discussion by alecgargett** [c3lang/c3c#1063] [General]
C3 appears to qualify to be added GitHub Linguist under the new criteria listed here:

https://github.com/github-linguist/linguist/issues/5756

as it has reached over 2000 files here (2.8K):
https://github.com/search?q=path%3A*.c3&type=Code&ref=advsearch&l=&l=

The instructions on how to do so are here:
https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#adding-a-language

Once it has been added it will automatically appear in other projects that use GitHub Linguist such as Languish:
https://tjpalmer.github.io/languish/

I'd love to be able to track C3's growth over time using Languish and it'd be nice for others to see C3 listed there too.

I'm not quite qualified to add C3 myself so I'd greatly appreciate if someone else with the knowhow were to do it.

**Comment by lerno** [comment]
I'd love to add it, but I don't think all of those are actually matching C3, and some are forks of the main repo. But it is getting there.
